metalus-core/src/main/scala/com/acxiom/pipeline/utils/ReflectionUtils.scala

```diff
@@ -368,4 +365,23 @@ object ReflectionUtils {
   } else {
     paramType <:< typeOf[Seq[_]]
   }
+
+  private def getStepPackage(objName: String, mirror: Mirror, pkg: Option[String], pipelineContext: PipelineContext): Option[String] = {
+    val stepPackages = pipelineContext.stepPackages.getOrElse(List())
+    if (pkg.isDefined && pkg.get.nonEmpty) {
+      val res = stepPackages.find(p => p == pkg.get)
```

Comment:
Remove this requirement. No need to only look at the ones on the step package list if the pkg is provided.